manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -44,12 +46,16 @@ spec:
           (omit .InfrastructureAnnotations "kubectl.kubernetes.io/last-applied-configuration" "gateway.istio.io/name-override" "gateway.istio.io/service-account" "gateway.istio.io/controller-version")
           (strdict "istio.io/rev" (.Revision | default "default"))
           (strdict
-            "ambient.istio.io/redirection" "disabled"
+            "ambient.istio.io/redirection" $redirection
             "prometheus.io/path" "/stats/prometheus"
             "prometheus.io/port" "15020"
             "prometheus.io/scrape" "true"
           ) | nindent 8 }}
       labels:
+        {{- if not $sandwich }}
```

Comment:
As-is breaks the template. The idea though, is that this results in `native_tunnel` being set for non-sandwich.
Waypoints that handle their own hbone are "native_tunnel". Sandwiched ones aren't. What do we think about that CP->zTunnel contract?